### ISSUE TITLE
batchnorm op

### ISSUE THREAD

**Issue by lolongcovas** [facebookincubator/AITemplate#92]
After looking at the examples of resnet, I didnt find a way to create batchnorm layer using `aitemplate.nn`. Should I have to asusme the nn.conv2d is a fusion of torch.nn.conv2d and torch.nn.BatchNorm2d in terms of op and weights?

**Comment by terrychenism** [comment]
Yes, BN params can be always fused into conv: https://github.com/facebookincubator/AITemplate/blob/main/examples/01_resnet-50/weight_utils.py#L59
